rsconnect/environment.py

```diff
@@ -6,38 +6,30 @@
 python -m rsconnect.environment
 ```
 """
-import collections
+from __future__ import annotations
+
 import datetime
 import json
 import locale
 import os
 import re
 import subprocess
 import sys
-
-try:
-    import typing
-    from typing import Optional
-except ImportError:
-    typing = None
```

Comment:
I removed this `try` statement because `typing` has been part of the standard library since Python 3.5.